mpf/tests/test_Snux.py

```diff
@@ -52,10 +52,10 @@ def test_ac_switch_and_pulse(self):
         c_ac_relay.enable = MagicMock()
         c_ac_relay.disable = MagicMock()
 
-        driver_11.pulse = MagicMock()
+        driver_11.pulse = MagicMock(return_value=0)
```

Comment:
Now that the code path uses the `wait_ms` response from `pulse()` to calculate switchover delays, need to ensure that a value is returned.